src/test/cbmc/Atomics/Stable/Exchange/main.rs

```diff
@@ -0,0 +1,38 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+use std::sync::atomic::{AtomicBool, Ordering};
+
+fn main() {
+    // pub fn compare_exchange(
```

Comment:
Is this comment intentional? Not seeing where `compare_exchange` is used.

Reply:
fixed